openfoam-interface/internal-interface/wrapper/simphonyInterface.C

```diff
@@ -945,6 +943,9 @@ void foam_modifyNumerics(std::string name, std::string fvSch, std::string fvSol,
         cDIS()
     );
 
+    TimeMod* t = (TimeMod*)(runTimes[name]);
```

Comment:
Almost but not! If you look that method, there is an "if" statment which ask if exists the file "system/controlDict". In the case of internal_interface wrapper we are avoiding to use the standard folder tree of any OF case, therefore we had to modify several classes, among them "fvSchemes", "fvSolution" and now "Time" to shortcut that control.